pom.xml

```diff
@@ -723,8 +724,9 @@
                         <configuration>
                             <includes>
                                 <include>**/RestQueryTranslationTest*.java</include>
+                                <include>**/io/confluent/ksql/rest/integration/**/*.java</include>
                             </includes>
-                            <forkCount>4</forkCount>
+                            <forkCount>3</forkCount>
```

Comment:
why decrease to 3 from 4? wouldnt we want to increase this if anything?

Reply:
@AlanConfluent can we have a Maven property for this value?  That'd give us room to set a default for local builds and override it for Jenkins builds.  Best of both worlds!

Reply:
Done! `multi-threaded-testing.forkCount`